plugins/cache.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./dist/plugins/cache");
```

Comment:
I think the path is wrong. Shouldn't it be:

```suggestion
module.exports = require("../dist/plugins/cache");
```

Could you test that?